gtk4/src/auto/widget.rs

```diff
@@ -75,11 +75,8 @@ pub trait WidgetExt: 'static {
     #[doc(alias = "gtk_widget_activate")]
     fn activate(&self) -> bool;
 
-    //#[doc(alias = "gtk_widget_activate_action")]
-    //fn activate_action(&self, name: &str, format_string: &str, : /*Unknown conversion*//*Unimplemented*/Fundamental: VarArgs) -> bool;
-
     #[doc(alias = "gtk_widget_activate_action_variant")]
```

Comment:
hmm, not great; it would have be nice to be able to set another doc_alias from gir I guess.

Reply:
https://github.com/gtk-rs/gir/issues/1067